exercises/E-strings-concatenation/exercise.js

```diff
@@ -1,3 +1,5 @@
 // Start by creating a variable `message`
-
+const myName = "Oksana Kostenko";
+const greeting = "Hello! My name is "
```

Comment:
Hi Oksana,
Line 3 seems to be missing the semi-colon ;

Reply:
Thanks Nigel, just fixed that